vendor/bat-native-ledger/src/bat/ledger/internal/ledger_impl.cc

```diff
@@ -737,13 +737,9 @@ void LedgerImpl::OnGrant(ledger::Result result,
 }
 
 void LedgerImpl::GetGrantCaptcha(
-    const std::vector<std::string>& headers) const {
-  bat_grants_->GetGrantCaptcha(headers);
-}
-
-void LedgerImpl::OnGrantCaptcha(const std::string& image,
-                                const std::string& hint) {
-  ledger_client_->OnGrantCaptcha(image, hint);
```

Comment:
I don't see this removed from `ledger_client.h` and then all other places. Do we use it somewhere else?

Reply:
OnGrantCaptcha is the callback I am passing down from rewards_service to ledger